### PR TITLE
[PagePartBundle] Make it possible to override the twig template for a custom PagePartWidget

### PR DESCRIPTION
**Pull request by jockri**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

File: src/Kunstmaan/PagePartBundle/Helper/FormWidgets/PagePartWidget.php [hunk 1]
```diff
@@ -128,11 +128,19 @@ public function getFormErrors(FormView $formView)
     }
 
     /**
+     * @param string $template
```
Comment (wimvds) [inline src/Kunstmaan/PagePartBundle/Helper/FormWidgets/PagePartWidget.php]: Why did you add these here?They are already in the FormWidget class (& interface)that this class inherits from...

**Comment by jockri** [comment]
@wimvds fixed it

**Comment by krispypen** [comment]
@roderik :+1:

**Comment by tarjei** [comment]
Simple question. If I understand this correctly, then this can be used to override the template shown in the Admin section of the site. What should I do in my code to use this?
